docs/tutorials/quickstart/quick-start-cli.md

```diff
@@ -34,7 +34,7 @@ Download and install a compiler.
 
 ###### 3. Install Mbed CLI
 
-To install Mbed CLI, run `pip install mbed-cli` from your command-line.
+To install Mbed CLI, run `sudo pip install mbed-cli` from your command-line.
```

Comment:
```suggestion
To install Mbed CLI, install in through `pip`. The specific invocation depends on your operating system:
 * On Windows, run `pip install mbed-cli` from your command-prompt.
 * On Mac, run `pip install --user mbed-cli` from your  command-line.
 * On Linux, run `sudo pip install mbed-cli` from your command-line.
```

Reply:
Because this is in the Windows section, do we need to state that it's operating system specific?
Also, @j3hill , I think you mentioned this change was for macOS. Should we move it to the macOS section?

Reply:
Ah, then there should be no need for `sudo` here. I did not realize that this was the windows section. Diffs can make that hard.